src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -55,7 +55,7 @@ class ErrorHandler
         \E_USER_DEPRECATED => 'User Deprecated',
         \E_NOTICE => 'Notice',
         \E_USER_NOTICE => 'User Notice',
-        \E_STRICT => 'Runtime Notice',
+        2048 => 'Runtime Notice', // E_STRICT
```

Comment:
I did this because of the solution being proposed in the third paragraph [here](https://wiki.php.net/rfc/deprecations_php_8_4#deprecate_dom_php_err_constant), but I like your solution better. Let's update

Reply:
I could not update `src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php`, because everything's static in here